changelog/bugfixes/2022-01-14-enable-icmpv6-router-adverts.md

```diff
@@ -0,0 +1 @@
+- network: Enable the ICMPv6 RAs to fix IPv6 address assignment ([PR#12](https://github.com/flatcar-linux/coreos-cloudinit/pull/12))
```

Comment:
This changelog entry won't be used, only the ones in coreos-overlay. Since all PRs are merged on the same time, I think one big entry there is best to describe the behavior change for bootengine, cloudinit, etc

Reply:
Yes, I will be writing a detailed changelog in the coreos-overlay when I create the commit id PR. Having a changelog here would be helpful to track in case we need project-based changes.